java/servers/src/org/xtreemfs/mrc/database/VolumeInfo.java

```diff
@@ -73,6 +73,28 @@
      * @return the volume's quota in bytes
      */
     public long getVolumeQuota() throws DatabaseException;
+
+    /**
+     * Returns the volume's voucher size in bytes.
+     * 
+     * @return the volume's voucher size in bytes.
+     */
+    public long getVolumeVoucherSize() throws DatabaseException;
+
+    /**
+     * Returns the default user quota in bytes.
+     * 
+     * @return the default user quota in bytes.
+     */
+    public long getVolumeDefaultUserQuota() throws DatabaseException;
```

Comment:
Remove "Volume" from method name (same for other methods in this class that are not related to the whole volume).